src/app/components/shared-components/addons-detail/addons-detail.tsx

```diff
@@ -8,18 +8,13 @@ import {
 } from '@patternfly/react-core';
 import { ZoneIcon } from '@patternfly/react-icons';
 
-type AddonsProps = Record<string, number>;
+type AddonsProps = {
```

Comment:
Is it really needed to create props outside the component?

Reply:
its is just specifying the type of props

Reply:
https://www.pluralsight.com/guides/defining-props-in-react-function-component-with-typescript its the standard way if we are using typescript